site/src/sphinx/server-annotated-service.rst

```diff
@@ -662,6 +732,75 @@ class-level decorators and method-level decorators.
     sb.annotatedService(new MyAnnotatedServiceClass(),
                         new MyGlobalDecorator1());      // order 1
 
+The first rule is as explained before. However, if your own decorator annotations and `@Decorator`_ annotations
+are specified in a mixed order like below, you need to clearly specify their order using ``order()`` attribute
+of the annotation. In the following example, you cannot make sure in what order they decorate the service
+because Java collects repeatable annotations like `@Decorator`_ into a single container annotation like
+`@Decorators`_ so it does not know the specified order between `@Decorator`_ and `@LoggingDecorator`_.
+
+.. code-block:: java
+
+    public class MyAnnotatedServiceClass {
+
+        @Get("/hello")
+        @Decorator(MyMethodDecorator1.class)
+        @LoggingDecorator
+        @Decorator(MyMethodDecorator2.class)
+        public HttpResponse hello() { ... }
+    }
+
+``order()`` attribute makes them in order. It will work only for class-level and method-level decorators.
+The default value of ``order()`` attribute is ``0`` and the decorators are sorted in an ascending order of
+``order()`` value.
+
+.. code-block:: java
+
+    public class MyAnnotatedServiceClass {
+
+        @Get("/hello")
+        @Decorator(value = MyMethodDecorator1.class, order = 1)
+        @LoggingDecorator(order = 2)
+        @Decorator(value = MyMethodDecorator2.class, order = 3)
+        public HttpResponse hello() { ... }
+    }
+
+    // Global-level decorators will not be affected by 'order'.
+    sb.annotatedService(new MyAnnotatedServiceClass(),
+                        new MyGlobalDecorator1());
+
+Even if you want to make a method-level decorator execute first, you can adjust decorator's order as follows.
```

Comment:
How about:

    Note that you can even make a method-level decorator executed before a class-level decorator
    by adjusting the ``order()`` attribute: